electrum/lnpeer.py

```diff
@@ -1055,6 +1041,29 @@ async def on_open_channel(self, payload):
             if not channel_type.complies_with_features(self.features):
                 raise Exception("sender has sent a channel type we don't support")
 
+        if self.is_channel_type():
+            is_zeroconf = channel_type & channel_type.OPTION_ZEROCONF
```

Comment:
two suggestions:
- use `ChannelType.OPTION_ZEROCONF` instead of `channel_type.OPTION_ZEROCONF`
- cast to bool for sanity - without that it is an int
```suggestion
            is_zeroconf = bool(channel_type & ChannelType.OPTION_ZEROCONF)
```

Reply:
Thanks, yeah this looks better. Fixed in cb56f0873dc65dccfd3111d1ca675bc8991a31bb